From 25e5bea89f4859d4e62c5170225e3ff0088a6401 Mon Sep 17 00:00:00 2001
From: Ruchi Kandoi <kandoiruchi@google.com>
Date: Thu, 3 Jan 2019 17:47:07 +0000
Subject: [PATCH 1/4] [Q_asb_2021-07] Revert "Tag detection does not stop after
 screen off"

This reverts commit 710575c54d9d35bc580757d91a46cf672d6fbda0.

Reason for revert: Causes a regression on NCI 1.0 devices

Change-Id: I80f5cf1fad45fdc4c7d74d369735758df17e7ace
(cherry picked from commit 83be4b6be65ce9e6e95b38c14a05829771bef59b)
---
 nci/jni/NativeNfcManager.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/nci/jni/NativeNfcManager.cpp b/nci/jni/NativeNfcManager.cpp
index 65f006f6..0fd921f9 100644
--- a/nci/jni/NativeNfcManager.cpp
+++ b/nci/jni/NativeNfcManager.cpp
@@ -364,11 +364,6 @@ static void nfaConnectionCallback(uint8_t connEvent,
       }
 
       nativeNfcTag_resetPresenceCheck();
-      if (!isListenMode(eventData->activated) &&
-          (prevScreenState == NFA_SCREEN_STATE_OFF_LOCKED ||
-           prevScreenState == NFA_SCREEN_STATE_OFF_UNLOCKED)) {
-        NFA_Deactivate(FALSE);
-      }
       if (isPeerToPeer(eventData->activated)) {
         if (sReaderModeEnabled) {
           DLOG_IF(INFO, nfc_debug_enabled) << StringPrintf(
-- 
2.34.1

